gsy_e_sdk/cli.py

```diff
@@ -93,25 +91,25 @@ def run(base_setup_path, setup_module_name, username, password, domain_name, web
     os.environ["API_CLIENT_RUN_ON_REDIS"] = "true" if run_on_redis else "false"
     if not run_on_redis:
         if simulation_config_path is not None:
-            config_file_path = create_simulation_config_path(base_setup_path, simulation_config_path)
-            config = read_simulation_config_file(config_file_path)
+            config = read_simulation_config_file(simulation_config_path)
```

Comment:
Loosely related to this PR. The exception raised in `read_simulation_config_file` mentions a `SIMULATION_CONFIG_FILE_PATH`: https://github.com/gridsingularity/gsy-e-sdk/blob/3dea1f42e283736ccbaaac6e51f4127e2bbfb020/gsy_e_sdk/utils.py#L273

Is this actually ever declared?

Reply:
You can declare this variable in your environment in order to bypass the config_file_path. If you do not provide one of them, this exception is raised.

Reply:
Could you please point me to the code where this is implemented? I can't seem to find it in the repos.

Reply:
Thanks for dealing with it 👍